lib/FusionInventory/Agent/Tools/Win32.pm

```diff
@@ -25,6 +25,7 @@ use Win32::TieRegistry (
     ArrayValues => 0,
     qw/KEY_READ/
 );
+use Win32API::Registry qw( :ALL );
```

Comment:
which perl module should I have for this one ?

Reply:
https://metacpan.org/pod/Win32API::Registry
if I remember, Win32::TieRegistry use this module, so it will works without install this module

Reply:
i've downloaded https://cpan.metacpan.org/authors/id/C/CH/CHORNY/Win32API-Registry-0.33.tar.gz, extracted it and put it in the lib folder, without success. The format is different than the other modules thought, so it's probably not what is expected

Reply:
[Win32API.tar.gz](https://github.com/fusioninventory/fusioninventory-agent/files/12302602/Win32API.tar.gz)

décompresses dans le dossier perl/vendor/lib/, tu devrait avoir un dossier `Win32API` et dedans un dossier `Registry` et un fichier `Registry.pm`

au pire, je peux t'envoyer tout mon dossier avec les modules